pkg/volume/azure_dd/azure_provision.go

```diff
@@ -143,7 +147,11 @@ func (p *azureDiskProvisioner) Provision() (*v1.PersistentVolume, error) {
 	// create disk
 	diskURI := ""
 	if kind == v1.AzureManagedDisk {
-		diskURI, err = diskController.CreateManagedDisk(name, skuName, requestGB, *(p.options.CloudTags))
+		resourceGroup := ""
+		if rg, found := p.options.PVC.Annotations[PVCAnnotationResourceGroup]; found {
+			resourceGroup = rg
+		}
+		diskURI, err = diskController.CreateManagedDisk(name, skuName, resourceGroup, requestGB, *(p.options.CloudTags))
```

Comment:
check whether p.options.CloudTags is nil?

Reply:
good catch, fixed